internal/drivers/k3s_in_docker/driver.go

```diff
@@ -254,6 +254,13 @@ configs:
 			cluster.Server = config.Host
 		}
 
+		// Rename the kube context to the name of the test harness.
+		// This makes life easier for someone interacting with the test cluster from their host machine
+		clog.DebugContext(ctx, "renaming kubeconfig context", "context", k.name)
+		kcfg.Contexts[k.name] = kcfg.Contexts["default"]
+		kcfg.CurrentContext = k.name
+		delete(kcfg.Contexts, "default")
```

Comment:
```suggestion
```

only suggestion, lets keep around the "default" just in case its useful for tooling